pg_lakehouse/README.md

```diff
@@ -0,0 +1,221 @@
+<h1 align="center">
+  <img src="../docs/logo/pg_lakehouse.svg" alt="pg_lakehouse" width="500px">
+<br>
+</h1>
+
+## Overview
+
+`pg_lakehouse` is an extension that transforms Postgres into an analytical query engine over data lakes like S3. Queries are pushed down to [Apache DataFusion](https://github.com/apache/datafusion), which significantly improves performance. Any combination of the following object stores, table formats, and file formats is supported.
+
+### Object Stores
+
+- [x] Amazon S3
+- [x] Local file system
+- [ ] Azure Blob Storage (coming soon)
+- [ ] Google Cloud Storage (coming soon)
+- [ ] HDFS (coming soon)
+
+### File Formats
+
+- [x] Parquet
+- [x] CSV
+- [x] JSON
+- [x] Avro
+- [ ] ORC (coming soon)
+
+### Table Formats
+
+- [x] Deltalake
+- [ ] Apache Iceberg (coming soon)
+
+## Motivation
+
+Today, an enormous amount of time and money is spent running and moving data into cloud data warehouses. At the same time, much of this data already lives in S3. `pg_lakehouse` allows companies to build a data warehouse on top of their existing Postgres and S3 with zero new infrastructure, data movement, loss of data freshness, or vendor lock-in.
+
+`pg_lakehouse` uses the foreign data wrapper (FDW) API to connect to data lakes and the executor hook API to push queries to DataFusion. While FDWs over S3 and Parquet exist in the Postgres extension ecosystem, these FDWs lack support for many file and table formats and are very slow for large analytical workloads. `pg_lakehouse` differentiates itself by supporting these formats and by being very fast.
```

Comment:
who are the target customers / users of this extension? are they customer trying to move to cloud? or customer who are already on the cloud? If the end goal is to run this extension on a managed database by cloud providers, implementing your own fdw might be duplicating the effort that cloud provider has already done. Essentially i think cloud providers already have their own in-house extension to support move data from datalake to database. 

They dont necessarily support moving data from one cloud provider to another. I think that could also be an interesting pool of customers.

Reply:
ah i see the problem is about the file / table format. Have you tried to do any feature request for cloud provider to enhance them?

Reply:
Not yet, but we are exploring that as well :)